apps/nextra/pages/en/build/move/move-on-aptos/object-code-deployment.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Object Code Deployment"
+---
+
+# Introduction
+This document goes through how you can deploy code to [Objects](./objects). This is the recommended way of deploying code to the blockchain, as this reduces deployment complexity,
+ and safely manages access control policies for the code owner.
+
+Deploying code to objects will guarantee the following:
+- Each deployment publishes to a new address.
+- Only the **owner of the code object** can upgrade and freeze the code.
+
+This means you can transfer the object to a new owner, and they will have full ownership of the code. You are granting them the rights to upgrade and freeze the code.
+There is no need to manage seeds, or deploy to a new address on each deployment. Object code deployment greatly simplifies the deployment process.
+
+## Instructions
+Below are the instructions on how to compile, deploy and upgrade code to objects.
+
+### 1. Compile code
+
+    1. Make sure `<your_module_name>` is left as a placeholder `_`. This is needed as the CLI command will override the address to specify
```

Comment:
to specify....?

Reply:
let me remove that.... Can just be
`This is needed as the CLI command will override the address.`